### PR TITLE
[release/v2.23] Patch release version bump, KKP update

### PR DESCRIPTION
**Pull request by pkprzekwas**
**What this PR does / why we need it**:
What the title says.

**What type of PR is this?**
/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by embik — CHANGES_REQUESTED** [review]

File: modules/api/go.mod [hunk 1]
```diff
@@ -67,7 +67,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v3 v3.0.1
 	k8c.io/kubeone v1.6.2
-	k8c.io/kubermatic/v2 v2.23.5-0.20231005111451-2808590653ba
+	k8c.io/kubermatic/v2 v2.23.5-0.20231013074658-4f07772499c4
```
[the remark this answers is not in the document]
Comment (pkprzekwas) [reply modules/api/go.mod:70]: Good point. Totally forgot about it.

File: modules/web/src/assets/config/changelog.json [hunk 2]
```diff
@@ -1,13 +1,29 @@
 {
-  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.23.md#v2234",
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.23.md#v2235",
   "entries": [
+    {
+      "category": "added",
+      "description": "Introduce `DisableAdminKubeconfig` flag in `KubermaticSettings` to disable the admin kubeconfig feature from dashboard."
```
[the remark this answers is not in the document]
Comment (pkprzekwas) [reply modules/web/src/assets/config/changelog.json:6]: AFAIK there is a switch in the dashboard.

**Review by Waseem826 — APPROVED** [review]
/approve

**Review by Waseem826 — APPROVED** [review]

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

**Pull request merged**